src/Nest/Mapping/Types/Specialized/Attachment/AttachmentProperty.cs

```diff
@@ -19,6 +20,7 @@ public interface IAttachmentProperty : IDocValuesProperty
 		ITextProperty LanguageField { get; set; }
 	}
 
+	[Obsolete("Removed in 6.0.")]
 	[DebuggerDisplay("{DebugDisplay}")]
 	public class AttachmentProperty : DocValuesPropertyBase, IAttachmentProperty
```

Comment:
`AttachmentProperty` can be removed altogether in 6.x - it's used for the mapper-attachment plugin in 5.x, which is superseded by the ingest-attachment plugin.